README.md

```diff
@@ -32,6 +32,40 @@ Since the project is under active development, we recommend watching the reposit
 regularly for updates. Detailed documentation and usage instructions will be provided as the project
 progresses.
 
+## Testing the Mender MCU Client (POSIX)
+
+### Dependencies
+- cmake
+- libcurl4-openssl-dev
+- libcjson-dev
+- libmbedtls-dev
+
+### Building the Client
+1. Configure the build:
+   ```
+   cmake -B build tests -DCONFIG_MENDER_PLATFORM_FLASH_TYPE="posix" -DCONFIG_MENDER_PLATFORM_LOG_TYPE="posix" -DCONFIG_MENDER_PLATFORM_NET_TYPE="generic/curl" -DCONFIG_MENDER_PLATFORM_SCHEDULER_TYPE="posix" -DCONFIG_MENDER_PLATFORM_STORAGE_TYPE="posix" -DCONFIG_MENDER_PLATFORM_TLS_TYPE="generic/mbedtls" -DCONFIG_MENDER_CLIENT_ADD_ON_INVENTORY=ON -DCONFIG_MENDER_CLIENT_ADD_ON_CONFIGURE=OFF -DCONFIG_MENDER_CLIENT_CONFIGURE_STORAGE=OFF -DCONFIG_MENDER_CLIENT_ADD_ON_TROUBLESHOOT=OFF -DCONFIG_MENDER_FULL_PARSE_ARTIFACT=ON -DCONFIG_MENDER_PROVIDES_DEPENDS=ON
```

Comment:
Specifying here the defines will soon get out of date. Suggestion: create a one liner wrapper script for this and call it both from here and from the CI job :bulb: